### DISCUSSION TITLE
Cut-cell method used for EB_CNS test (C++ version of EB/CNS tutorial)

### DISCUSSION THREAD

**Discussion by ashwathsv** [AMReX-Codes/amrex#2493] [Q&A]
Hi! What cut-cell method does the Tests/EB_CNS code use? Is it the same method as the equivalent Fortran code or a different one? Any references to the methods used would be helpful. Thanks!

**Comment by WeiqunZhang** [comment]
Part of the algorithm is discussed in https://doi.org/10.1016/j.jcp.2022.111315.
